UnitystationLauncher/Views/SecurityPanelView.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             FontSize="12"
+             x:Class="UnitystationLauncher.Views.SecurityPanelView">
+    <DockPanel>
+        <StackPanel>
+            <Button Width="160" Command="{Binding OnScan}">Scan</Button>
+        </StackPanel>
+        <StackPanel>
+            <Button Width="160" Command="{Binding OnUpdateSafe}">Update Safe</Button>
```

Comment:
A better label on this button would be helpful, not sure what this does by reading the name.

Reply:
yeah is debug